src/index.js

```diff
@@ -95,6 +97,27 @@ var client = function (sitekey, privatekey) {
     );
   };
 
+  /**
+   * Put a sentiment click value to the conversational search interactions API
+   *
+   * @param {string} conversationId  Argument 1: Conversation ID
+   * @param {('positive'|'negative'|'neutral')} sentimentValue  Argument 2: Sentiment value (positive, negative, neutral)
+   * @returns {Promise<boolean|Object>} Resolves to true if successful, rejects with an error object if unsuccessful
+   */
+  this.putSentimentClick = function (conversationId, sentimentValue) {
+    return putSentimentClick(this.apiHostname, this.sitekey, conversationId, sentimentValue);
+  };
+
+  /**
+   * Post a copy click value to the conversational search interactions API
+   *
+   * @param {string} conversationId  Argument 1: Conversation ID
+   * @returns {Promise<boolean|Object>} Resolves to true if successful, rejects with an error object if unsuccessful
+   */
+  this.putCopyClick = function (conversationId) {
+    return putCopyClick(this.apiHostname, this.sitekey, conversationId);
```

Comment:
`putCopyClick` is not imported yet, maybe add TODO to remind ourselves later

Reply:
jup, I think I will just remove for now